nix/default.nix

```diff
@@ -0,0 +1,48 @@
+{ system ? builtins.currentSystem }:
+let
+  pkgs = (import ./nixpkgs.nix {
+    overlays = [ (final: prev: {
+      pcre = prev.pcre.overrideAttrs (x: {
+        configureFlags = x.configureFlags ++ [ "--enable-static" ];
+      });
+    })];
+    config = {
+      packageOverrides = pkg: {
+        glib = pkg.glib.overrideAttrs(x: {
+          outputs = [ "bin" "out" "dev" ];
+          mesonFlags = [
+            "-Ddefault_library=static"
+            "-Ddevbindir=${placeholder ''dev''}/bin"
+            "-Dgtk_doc=false"
+            "-Dnls=disabled"
+          ];
+        });
+        systemd = pkg.systemd.overrideAttrs(x: {
+          mesonFlags = x.mesonFlags ++ [ "-Dstatic-libsystemd=true" ];
+          postFixup = ''
+            ${x.postFixup}
+            sed -ri "s;$out/(.*);$nukedRef/\1;g" $lib/lib/libsystemd.a
+          '';
+        });
+      };
+    };
+  });
+
+  self = with pkgs; {
+    conmon-static = conmon.overrideAttrs(x: {
+      name = "conmon-static";
+      src = ./..;
+      doCheck = false;
```

Comment:
I think we can remove that:
```suggestion
```

Reply:
Seem this is not useful? https://nixos.org/nixpkgs/manual/#ssec-check-phase

Reply:
We do not execute checks on conmon build, so the flag seems not necessary to me, right?

Reply:
removed ;-)

Reply:
oic why did I add it... in case of crun `doCheck = false;` is required...

For simple stupid and sync, I just copy-and-paste `doCheck = false;` for all PR ;-)